build/amis/README.md

```diff
@@ -121,9 +131,49 @@ packer build -var-file base-images-us-east-1.json -var ami_regions='us-west-2'
 
 Connect remotely to an instance created from the image and run the Node Conformance tests using the following commands:
 
+### Initialize a CNI
+
+As root:
+
+(copied from [containernetworking/cni](https://github.com/containernetworking/cni#how-do-i-use-cni))
+
+```sh
+mkdir -p /etc/cni/net.d
+wget -q https://github.com/containernetworking/cni/releases/download/v0.6.0/cni-amd64-v0.6.0.tgz
+tar -xzf cni-amd64-v0.6.0.tgz --directory /etc/cni/net.d
+cat >/etc/cni/net.d/10-mynet.conf <<EOF
+{
+    "cniVersion": "0.2.0",
+    "name": "mynet",
+    "type": "bridge",
+    "bridge": "cni0",
+    "isGateway": true,
+    "ipMasq": true,
+    "ipam": {
+        "type": "host-local",
+        "subnet": "10.22.0.0/16",
+        "routes": [
+            { "dst": "0.0.0.0/0" }
+        ]
+    }
+}
+EOF
+cat >/etc/cni/net.d/99-loopback.conf <<EOF
+{
+    "cniVersion": "0.2.0",
+    "name": "lo",
+    "type": "loopback"
+}
+EOF
+```
+
+### Run the e2e node conformance tests
+
+As a non-root user:
+
 ```sh
 wget https://dl.k8s.io/$(< /etc/kubernetes_community_ami_version)/kubernetes-test.tar.gz
 tar -zxvf kubernetes-test.tar.gz kubernetes/platforms/linux/amd64
 cd kubernetes/platforms/linux/amd64
-sudo ./ginkgo --nodes=8 --flakeAttempts=2 --focus="\[Conformance\]" --skip="\[Flaky\]|\[Serial\]|\[sig-network\]|Container Lifecycle Hook" ./e2e_node.test -- --k8s-bin-dir=/usr/bin
-```
\ No newline at end of file
+sudo ./ginkgo --nodes=8 --flakeAttempts=2 --focus="\[Conformance\]" --skip="\[Flaky\]|\[Serial\]|\[sig-network\]|Container Lifecycle Hook" ./e2e_node.test -- --k8s-bin-dir=/usr/bin --container-runtime=remote --container-runtime-endpoint unix:///var/run/containerd/containerd.sock --container-runtime-process-name /usr/local/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd\"]}"
```

Comment:
Now that a CNI is being initialized, we can probably remove the skip for `\[sig-network\]` and `Container Lifecycle Hook`, since those were previously added to work around the lack of CNI :)